sysid-application/src/main/native/cpp/view/AnalyzerPlot.cpp

```diff
@@ -7,38 +7,29 @@
 #include <algorithm>
 #include <cmath>
 #include <mutex>
-#include <thread>
-#include <vector>
 
 #include <fmt/format.h>
-#include <imgui.h>
-#include <implot.h>
 #include <units/math.h>
-#include <units/time.h>
 
 #include "sysid/Util.h"
-#include "sysid/analysis/AnalysisManager.h"
-#include "sysid/analysis/AnalysisType.h"
 #include "sysid/analysis/ArmSim.h"
 #include "sysid/analysis/ElevatorSim.h"
 #include "sysid/analysis/FilteringUtils.h"
 #include "sysid/analysis/SimpleMotorSim.h"
+#include "sysid/view/WindowLayout.h"
 
 using namespace sysid;
 
 static ImPlotPoint Getter(void* data, int idx) {
   return static_cast<ImPlotPoint*>(data)[idx];
 }
 
-static double simSquaredErrorSum = 0.0;
-static double squaredVariationSum = 0.0;
-static int timeSeriesPoints = 0;
-
 template <typename Model>
 static std::vector<std::vector<ImPlotPoint>> PopulateTimeDomainSim(
     const std::vector<PreparedData>& data,
-    const std::array<units::second_t, 4>& startTimes, size_t step,
-    Model model) {
+    const std::array<units::second_t, 4>& startTimes, size_t step, Model model,
+    double& simSquaredErrorSum, double& squaredVariationSum,
+    int& timeSeriesPoints) {
```

Comment:
If we're modifying variables, shouldn't we be using pointers or does that not apply to primitives?